.travis.yml

```diff
@@ -0,0 +1,12 @@
+sudo: false
+
+language: node_js
+
+node_js:
+  - "0.12"
+  - "4"
+  - "5"
```

Comment:
We aren't testing against 5 in our other agents. Is there something special about error reporting that needs 5 integration?